App/Sagas/Migration.ts

```diff
@@ -204,12 +204,11 @@ export function * announcePeer() {
   if (previousItems.length) {
     const previous = previousItems[0]
     const peerId = yield select(getPeerId)
-    const address = yield select(getAddress)
+    const contactInfo: ContactInfo = yield call(contact, peerId)
```

Comment:
Here we call the go layer's `contact()` method with our own peer id, so it returns a `ContactInfo` for the local peer. That is what we announce and send to the server.